### ISSUE TITLE
Embed node/npm with the application

### ISSUE THREAD

**Issue by dsyme** [fable-compiler/Fable#137]
Building from clean on Windows I get

```
cmd /C npm version 0.2.14
'npm' is not recognized as an internal or external command,
```

Should 'npm' be downloaded as a nuget package, e.g. from here?

```
https://www.nuget.org/packages/Npm/
```

thanks
don

**Comment by alfonsogarciacaro** [comment]
I gave this a try before. The problem with that package is it only works for Windows x64 machines and prevents Fable from being used on Mac and Linux.

Anyways, I've received other request to allow Fable's usage without having to install Node (I do JS development and I just took that for granted) so I'm planning now to add a FAKE target to package the program with [the proper Node binary for the platform](https://nodejs.org/en/download/). That'll make it easier to embed Fable in other applications.
